phpstan.neon

```diff
@@ -60,7 +60,15 @@ parameters:
     excludePaths:
         - src/cli/src/Flow/CLI/Command/*
         - src/core/etl/src/Flow/ETL/Formatter/ASCII/ASCIITable.php
+        - src/core/etl/src/Flow/ETL/Function/HTMLQuerySelectorAll.php
+        - src/core/etl/src/Flow/ETL/Function/HTMLQuerySelector.php
+        - src/core/etl/src/Flow/ETL/Row/Entry/HTMLEntry.php
         - src/core/etl/src/Flow/ETL/Sort/ExternalSort/RowsMinHeap.php
+        - src/core/etl/tests/Flow/ETL/Tests/Integration/Function/HTMLQuerySelectorAllTest.php
+        - src/core/etl/tests/Flow/ETL/Tests/Integration/Function/HTMLQuerySelectorTest.php
+        - src/core/etl/tests/Flow/ETL/Tests/Unit/Function/HTMLQuerySelectorAllTest.php
+        - src/core/etl/tests/Flow/ETL/Tests/Unit/Function/HTMLQuerySelectorTest.php
+        - src/core/etl/tests/Flow/ETL/Tests/Unit/Row/Entry/HTMLEntryTest.php
```

Comment:
that's not great, can't we suppress only specific errors? Those paths are also prevent static analysis from doing other checks on those classes. 

+ if you keep it this way and we drop support for 8.2 and 8.3 we wont get even notified that those "exclusions" are not longer needed. 

When you suppress only specific errors, when there is nothing to suppress phpstan will notify us that given suppression is no longer needed.

Reply:
Here is a dedicated commit doing that: [`0156e72` (#1960)](https://github.com/flow-php/flow/pull/1960/commits/0156e720a9c9cf6d65e50ae9fa34faeb4bcd5878)

Reply:
Removed tests from exclusion, and left only dedicated classes.